app/src/main/res/layout/activity_help_legend.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I do in general like the way HelpActivity looks. It seems a little bit empty, try spreading out the spacing between the icons in the legend, that might help. This screen is a little harsh on the eyes with all the white but we'll talk about colour/icons later.